shared/types/response-v3.ts

```diff
@@ -115,3 +115,48 @@ export type AttachmentFieldResponseV3 = {
   hasBeenScanned: boolean
   answer: string
 }
+
+/**
+ * AttachmentResponses with additional server injected metadata on email and storage v2+ forms.
+ */
+export type ParsedClearAttachmentFieldResponseV3 = AttachmentFieldResponseV3 & {
+  filename: string
+  content: Buffer
+}
+
+/**
+ * AttachmentResponses with hydrated content from server
+ *
+ * Note: JSON.parse(JSON.stringify(fooBuffer)) does not reconstruct the Buffer object
+ */
+export type HydratedClearAttachmentFieldResponseV3 =
+  AttachmentFieldResponseV3 & {
+    filename: string
+    content: {
+      data: string
+      type: 'buffer'
+    }
+  }
```

Comment:
Can clarify here what the meaning of "Hydrated" is? haha. Which part of the backend hydrates the attachment

Reply:
https://github.com/opengovsg/formsg-javascript-sdk/blob/develop/src/crypto-v3.ts#L76

I'm referring to hydration (since parsed is already used) to be `JSON.parse(foo)`

Reply:
Im thinking if there's a more elegant way to handle your comment [below](https://github.com/opengovsg/FormSG/pull/7181#discussion_r1546018740). This is merely an artefact from it.

Reply:
Removed in 247077854